node/src/actors/chain_manager/mod.rs

```diff
@@ -1290,6 +1290,8 @@ impl ChainManager {
                         consensus_constants.superblock_committee_decreasing_step,
                         chain_info.highest_superblock_checkpoint.checkpoint,
                         superblock_index,
+                        (chain_info.consensus_constants.collateral_age
+                            + chain_info.consensus_constants.activity_period)/superblock_period
```

Comment:
I would prefer to have this logic defined in a separate function. Something like

```rust
fn last_superblock_signed_by_bootstrap(consensus_constants: &ConsensusConstants) -> u32 {}
```

And reuse the same variable when calculating the `reputed_ars_members`.